components/infobox/wikis/pubg/infobox_league_custom.lua

```diff
@@ -0,0 +1,223 @@
+---
+-- @Liquipedia
+-- wiki=pubg
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local Template = require('Module:Template')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local PrizePoolCurrency = require('Module:Prize pool currency')
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _TODAY = os.date('%Y-%m-%d', os.time())
+local _GAME = mw.loadData('Module:GameVersion')
+
+local _MODES = {
+	solo = 'Solos[[Category:Solos Mode Tournaments]]',
+	duo = 'Duos[[Category:Duos Mode Tournaments]]',
+	squad = 'Squads[[Category:Squads Mode Tournaments]]',
+	['2v2'] = '2v2 TDM[[Category:2v2 TDM Tournaments]]',
+	['4v4'] = '4v4 TDM[[Category:4v4 TDM Tournaments]]',
+	['war mode'] = 'War Mode',
+	default = '[[Category:Unknown Mode Tournaments]]',
+}
+_MODES.solos = _MODES.solo
+_MODES.duos = _MODES.duo
+_MODES.squads = _MODES.squad
+_MODES.tdm = _MODES['2v2']
+
+local _PERSPECTIVES = {
+	fpp = {'FPP'},
+	tpp = {'TPP'},
+	mixed = {'FPP', 'TPP'},
+}
+_PERSPECTIVES.first = _PERSPECTIVES.fpp
+_PERSPECTIVES.third = _PERSPECTIVES.tpp
+
+local _PLATFORMS = {
+	pc = '[[PC]][[Category:PC Competitions]]',
+	mobile = '[[Mobile]][[Category:Mobile Competitions]]',
+	newstate = '[[New State]][[Category:Mobile Competitions]]',
+	pclite = '[[PC LITE]][[Category:PC Competitions]][[Category:PC LITE Competitions]]',
+	peace = '[[Peacekeeper Elite|Peace Elite]][[Category:Peacekeeper Elite Competitions]][[Category:Mobile Competitions]]',
+	bgmi = '[[Battlegrounds Mobile India|BGMI]][[Category:BGMI Competitions]][[Category:Mobile Competitions]]',
```

Comment:
you sure about using `BGMI Competitions` over `Battlegrounds Mobile India Competitions` for the category?
it is a change compared to the current infobox league